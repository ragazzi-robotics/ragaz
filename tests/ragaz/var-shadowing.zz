var a = 123


def foo(a: str):
    print(a)
    var a: int = 1
    if True:
        var a: float = 2.0
        print(a)
    print(a)


def main():
    print(a)
    foo("abc")
