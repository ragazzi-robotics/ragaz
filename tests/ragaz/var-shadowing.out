123
abc
2.000000
1
